test/test_secret_manager_with_cli.py

```diff
@@ -30,6 +30,8 @@ def test_secret_manager(base_url, headers):
 
     createSecret.createSecret(base_url, "secretData2", "secret002", None, "mysecret", "20d")
 
+    putSecretValue.putSecrectValue(base_url, "secret001", "secretData3")
```

Comment:
seems the param order not make sense, generally, if I'm a user for the secret, i will use it by the following cmd:

putSecretVaulle --secretName "xxxx" --secretData "xxx"

same with other APIs

Reply:
ok, I will submit a new PR later to modify the parameter order of the create secret test